devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -61,18 +63,57 @@ metadata {
 		main "motion"
 		details(["motion", "battery", "tamper"])
 	}
-}
+
+	// Preferences for Everspring SP815 and Everspring SP817
+	// preferences {
+		section {
+			input(
+					title: "Settings Available For Everspring SP815 and SP817 only",
+					description: "",
+					type: "paragraph",
+					element: "paragraph"
+			)
+			input (
+					title: "Temperature and Humidity Auto Report (Everspring SP815 only):",
+					description: "This setting allows to adjusts report time (in seconds) of temperature and humidity report.",
+					name: "temperatureAndHumidityReport",
+					type: "number",
+					range: "600..1440",
+					defaultValue: 600
+			) // defaultValue: 600
+			input (
+					title: "Re-trigger Interval Setting (Everspring SP815 and Everspring SP817 only):",
+					description: "The setting adjusts the sleep period (in seconds) after the detector has been triggered. No response will be made during this interval if a movement is presented. Longer re-trigger interval will result in longer battery life.",
+					name: "retriggerIntervalSettings",
+					type: "number",
+					range: "10..3600",
+					defaultValue: 180
+			) // defaultValue: 180
+		}
+	}
 
 def installed() {
 // Device wakes up every 4 hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
 	sendEvent(name: "tamper", value: "clear", displayed: false)
-	response(initialPoll())
 }
 
 def updated() {
+	log.debug "updated"
 // Device wakes up every 4 hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	response(initialPoll())
```

Comment:
you shouldn't need to have `initialPoll` here and in `configure`

Reply:
ok, done